libyul/YulString.h

```diff
@@ -79,6 +85,11 @@ class YulStringRepository: boost::noncopyable
 		return hash;
 	}
 	static constexpr std::uint64_t emptyHash() { return 14695981039346656037u; }
+	void reset()
```

Comment:
Please document that this should be used with utmost care.

Reply:
... it can only be used if there are no dangling uses of `YulString` anywhere in memory.